### DISCUSSION TITLE
Assets problem on Chrome when minnification is activated

### DISCUSSION THREAD

**Discussion by yo5bdm** [wintercms/winter#162] [Q&A] [answered]
I have a weird problem regarding the minnification process. Whe it is enabled from the cms.php file ('enableAssetMinify' => true,) the platform does not work on chrome. Weirdly enough, on firefox it works just fine.

Google is not helping me with this matter and I'd love to keep it enabled as my loading time drops from 600ms to about 250ms.

**Comment by mjauvin** [comment, marked as answer]
Can you provide more details as this is working fine for me.

Can you try disabling all plugin and see if this is still misbehaving for you?
